detection/notebooks/00_webcam.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>In second sentence, need to change "ImageNet" to "COCO"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>